site/content/blog/2022-05-01-whats-new-in-svelte-may-2022.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "What's new in Svelte: May 2022"
+description: "Dynamically switch between HTML element types with `<svelte:element>`"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With yesterday's Svelte Summit behind us, we've got a lot of news to share! check out all of the recordings on the [Svelte Society YouTube Channel](https://www.youtube.com/sveltesociety) and the rest of this month's updates below...
+
+## What's new in Svelte
+- The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when rich text content from a CMS. If the tag is changed, the children will be preserved unless there's a transition attached to the element. Check out the [docs](https://svelte.dev/docs#template-syntax-svelte-element) or the [tutorial](https://svelte.dev/tutorial/svelte-element) for more info (**3.47.0**)!
+
+
+## Language Tools updates
+- `svelte:element` and `sveltekit:reload` are now supported
+- Invalid Svelte import paths will now be automatically detected - see PR for getting back the old behavior ([#1448](https://github.com/sveltejs/language-tools/pull/1448))
+- `source.sortImports` lets you sort imports without deleting unused imports ([#1338](https://github.com/sveltejs/language-tools/issues/1338))
+- Hovering over HTML attributes will now show HTML hover info instead of the TS hover info - resulting in much more useful information ([#1447](https://github.com/sveltejs/language-tools/pull/1447))
+
+
+## What's new in SvelteKit
+- Files and directories can now be named `__tests__` and `__test__` in the routes directory ([#4438](https://github.com/sveltejs/kit/pull/4438))
```

Comment:
- Named layouts https://github.com/sveltejs/kit/pull/4388
- Support for Netlify Edge Functions https://github.com/sveltejs/kit/pull/4657
- Support for Vercel build output API https://github.com/sveltejs/kit/pull/4663
- Custom `load` dependencies https://github.com/sveltejs/kit/pull/4536
- `sveltekit:reload` https://github.com/sveltejs/kit/pull/4545

Reply:
I captured named layouts in the breaking changes. Captured the rest in my next commit :)